ci-operator/step-registry/servicemesh/kiali/ossm-cypress-tests/execute/servicemesh-kiali-ossm-cypress-tests-execute-commands.sh

```
@@ -48,13 +48,18 @@ fi
 # download hack from kiali repo
 hack/download-hack-scripts.sh --kiali-branch ${KIALI_VERSION}
+# put hack folder to ../ where tests expect it
+cp -r _output/ ../_output
+
 # download prometheus script
 wget -N https://raw.githubusercontent.com/kiali/kiali/${KIALI_VERSION}/hack/use-openshift-prometheus.sh -O _output/kiali/hack/use-openshift-prometheus.sh
 chmod +x _output/kiali/hack/use-openshift-prometheus.sh
 
 # remove v from ISTIO version if there is any
 [[ $ISTIO_SAMPLE_APP_VERSION == v* ]] && ISTIO_SAMPLE_APP_VERSION="${ISTIO_SAMPLE_APP_VERSION#v}" || ISTIO_SAMPLE_APP_VERSION="$ISTIO_SAMPLE_APP_VERSION"
 _output/kiali/hack/istio/download-istio.sh -iv ${ISTIO_SAMPLE_APP_VERSION}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)